cfg/VideoStream.cfg

```diff
@@ -29,6 +29,8 @@ gen.add("hue", double_t, LEVEL.RUNNING, "Target hue", 0.5, 0.0, 1.0)
 gen.add("saturation", double_t, LEVEL.RUNNING, "Target saturation", 0.64, 0.0, 1.0)
 gen.add("auto_exposure", bool_t, LEVEL.RUNNING, "Target auto exposure", True)
 gen.add("exposure", double_t, LEVEL.RUNNING, "Target exposure", 0.5, 0.0, 1.0)
+gen.add("auto_focus", bool_t, LEVEL.RUNNING, "Target auto focus", True)
+gen.add("focus", double_t, LEVEL.RUNNING, "Target focus", 0.5, 0.0, 1.0)
```

Comment:
Is setting a default value to 0.5 reasonable? I don't have a camera with auto focus feature, so just a question.

Reply:
That's a good question. 0.5 is the value to set for autofocus to work. The range seems to depend on the camera. I'm wondering if there are cameras with software defined focus and without autofocus. If so, the value is relevant.

Reply:
I think 0.5 is reasonable, as it is in the middle of the range. Also, autofocus is enable per default, which I think is the default in OpenCV when nothing is set.